osmaxx-py/osmaxx/excerptexport/views.py

```diff
@@ -136,16 +137,38 @@ def list_orders(request):
 
 
 def access_denied(request):
+    view_context = {
+        'next_page': request.GET['next']
+    }
+    return render_to_response('excerptexport/templates/access_denied.html', context=view_context,
+                              context_instance=RequestContext(request))
+
+
+@login_required()
+def request_access(request):
     user_administrator_email = settings.OSMAXX['ACCOUNT_MANAGER_EMAIL']
     if not user_administrator_email:
         logging.exception(
             "You don't have an user account manager email address defined. Please set OSMAXX_ACCOUNT_MANAGER_EMAIL."
         )
 
-    view_context = {
-        'next_page': request.GET['next'],
-        'user': request.user,
-        'user_administrator_email': user_administrator_email,
-    }
-    return render_to_response('excerptexport/templates/access_denied.html', context=view_context,
-                              context_instance=RequestContext(request))
+    try:
+        email_message = _(
+            '{first_name} {last_name} requests access for Osmaxx. Please activate the user {username}.'
+        ).format(
+            username=request.user.username,
+            first_name=request.user.first_name,
+            last_name=request.user.last_name
+        )
+        send_mail('Request access for Osmaxx', email_message, request.user.email,
+                  [user_administrator_email], fail_silently=True)
+        messages.success(
+            request,
+            _('Your access request has been sent successfully. You will receive an email when your account is ready.')
+        )
+    except:
+        logging.exception(
+            "Sending access request e-mail failed! ("+email_message+")"
+        )
```

Comment:
Shouldn't the _user_ get notified in this case, too? (Error page or via on-site message.)